tools/buildsys/src/args.rs

```diff
@@ -92,6 +92,9 @@ pub(crate) struct Common {
 
     #[arg(long, env = "TWOLITER_TOOLS_DIR")]
     pub(crate) tools_dir: PathBuf,
+
+    #[arg(long, env = "BUILDSYS_CICD_HACK")]
+    pub(crate) cicd_hack: Option<String>,
```

Comment:
I think I'm OK with this variable name so long as we document its purpose in the code as well.

That said, it is a great hack 😄